uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/DAO/AbsenceType.php

```diff
@@ -479,6 +484,19 @@ static function &fields() {
           'bao' => 'CRM_HRLeaveAndAbsences_DAO_AbsenceType',
           'localizable' => 0,
         ) ,
+        'category' => array(
```

Comment:
Were this changes manually generated or done by the Build DAO script? I was expecting to see a change in the GenCodeChecksum at the top of the file.
The instructions are here: https://compucorp.atlassian.net/wiki/spaces/PCHR/pages/101089347/Creating+and+Updating+entities+BAO+DAO+for+an+extension